packages/cubejs-schema-compiler/src/adapter/PreAggregations.ts

```diff
@@ -1,40 +1,109 @@
+// @ts-nocheck
```

Comment:
Is it necessary here?

Reply:
Nope! I was using that during debugging... Good catch! I'll remove it of course!